en/controllers/components/pagination.rst

```diff
@@ -208,6 +208,48 @@ elements and URLs for pagination.
 
 .. versionadded:: 3.3.0
     Multiple Pagination was added in 3.3.0
+    
+Paginating the Same Model multiple Times
+----------------------------------------
+
+To paginate the same model multiple times within a single controller action you
+need to define an alias for the model::
+
+    // In a controller action
+    $this->paginate = [
+        'ArticlesTable' => [
+            'scope' => 'published_articles',
+            'limit' => 10,
+            'order' => [
+                'id' => 'desc',
+            ],
+        ],
+        'UnpublishedArticlesTable' => [
+            'scope' => 'unpublished_articles',
+            'limit' => 10,
+            'order' => [
+                'id' => 'desc',
+            ],
+        ],
+    ];
+    
+    TableRegistry::config('UnpublishedArticles', [
+        'className' => 'App\Model\Table\ArticlesTable',
+        'table' => 'articles',
+        'entityClass' => 'App\Model\Entity\Article',
+    ]);
```

Comment:
Should this be in the controller? Or should we tell the reader to place it in the app bootstrap or in the relevant model file?

Reply:
Good question. As I needed this for a single controller action/view I preferred placing it within this action so I clearly know what it was used for.

Reply:
ok.

Reply:
Should there be a comment above this call explaining what it does? I don't think `TableRegistry::config()` is a frequently used method.

Reply:
Has been solved by the latest commits already

Reply:
"Should there be a comment above this call explaining what it does?"
Dont see that added yet.

Reply:
:) should have read more carefully